tests/test_scheduler.py

```
@@ -1,192 +1,172 @@
 import pytest
-from unittest.mock import AsyncMock, MagicMock, patch, call
+from unittest.mock import AsyncMock, MagicMock, patch  # noqa: F401
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)